components/bal/numero-editor.js

```diff
@@ -115,6 +115,7 @@ function NumeroEditor({initialVoieId, initialValue, hasPreview, closeForm}) {
       setValidationMessages(validationMessages)
 
       await reloadNumeros()
+      reloadVoies()
```

Comment:
Il n'est nécessaire de recharger les voies que si une nouvelle voie vient d'être créer. On peut isoler ce cas pour éviter des recharger inutilement
```suggestion
      if (initialVoieId !== voie__id) {
          reloadVoies()
      }
```